lib/Notes/NotesAccordion/NotesAccordion.js

```diff
@@ -107,16 +128,20 @@ export default class NotesAccordion extends Component {
           />
         </Accordion>
 
-        <NotesAssigningModal
-          open={modalIsOpen}
-          onReset={onResetSearchResults}
-          onSave={this.onSaveAssigningResults}
-          onSearch={onSearch}
-          onClose={this.onCloseModal}
-          onSort={onSortDomainNotes}
-          onNeedMore={onNeedMoreDomainNotes}
-          notes={domainNotes}
-        />
+        {
+          modalIsOpen && (
+            <NotesAssigningModal
+              open
```

Comment:
I see that you intentionally made it this way, so I guess there are reasons for this. But I'm just curious what is the reason for having the check if `modalIsOpen` is `true` instead of doing:
```js 
<NotesAssigningModal
  open={modalIsOpen}
/>
```
And I think that with this change, the `open` prop of `NotesAssigningModal` is redundant since it's always true and the rendering of  `NotesAssigningModal` depends on `modalIsOpen`.